src/lib/components/composites/list/NamedList.svelte

```diff
@@ -97,11 +139,21 @@ This can be e.g. a search bar.
             <span class="text-hint italic">{emptyHint}</span>
         {:else}
             <ul class="scroll-box space-y-4 pb-1">
-                {#each loadedItems as item (keySelector(item))}
-                    <li>
-                        {@render listItemComponent?.(item)}
-                    </li>
-                {/each}
+                {#if groupSelector !== undefined}
+                    {#each Object.entries(groupBy(loadedItems, groupSelector)) as [key, values] (key)}
+                        <div class="space-y-1">
+                            {#await groupLabels}
+                                <h3 class="italic">Loading</h3>
```

Comment:
```suggestion
                                <h1 class="italic">Loading...</h1>
```
What do you think about this?

Reply:
I thought we agreed on not using "..." when loading something? But if you want this for clarity I can add them.

Reply:
That is true. Since the loading is italic it should be enough then.